node/src/stream_handler_pool.rs

```diff
@@ -488,7 +488,7 @@ impl StreamHandlerPool {
         let packet_len = packet.data.len();
         match tx_box.unbounded_send(packet) {
             Err(e) => {
-                debug!(
+                error!(
```

Comment:
Check and make sure there's a test that exercises this. If there is, it should assert on this `error!()` log. If there isn't, one should be written.